== SUBMITION (2019-03-31 15:21:26) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = int(input("r"))
a = int(input("r:A = math.pir2"))
v = int(input("r:V = 4/3math.pir3"))
print(round(r))
-- EXECUTION TIME:
0.216792
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 4, in <module>
    r = int(input())
ValueError: invalid literal for int() with base 10: '4.5'

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:11:56) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = input("r")
area = math.pi * (r ** 2)
volume = 4 / 3 * math.pi * (r ** 3)
print(round(area,3))


-- EXECUTION TIME:
0.159897
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math.pi * (r ** 2)
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:12:22) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = input("r")
area = math.pi * (r ** 2)
volume = 4 / 3 * math.pi * (r ** 3)
print(round(area,3))
print(round(volume,3)


-- EXECUTION TIME:
0.178866
-- ERROR:
  File "/home/codebench/codes/1590/825/main.py", line 10
    
    ^
SyntaxError: unexpected EOF while parsing

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:12:29) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = input("r")
area = math.pi * (r ** 2)
volume = 4 / 3 * math.pi * (r ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.158288
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math.pi * (r ** 2)
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:12:37) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = input("r")
area = math.pi * (r ** 2)
volume = 4 / 3 * math.pi * (r ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.173783
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math.pi * (r ** 2)
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:12:38) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = input("r")
area = math.pi * (r ** 2)
volume = 4 / 3 * math.pi * (r ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.191549
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math.pi * (r ** 2)
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:13:22) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = input("r")
area = math.pi * (r ** 2)
volume = 4 / 3 * math.pi * (r ** 3)
print(round(area, 3))
print(round(volume, 3))


-- EXECUTION TIME:
0.190397
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math.pi * (r ** 2)
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:55:07) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = float(input("r"))
area = math.pi * (r ** 2)
volume = 4 / 3 * math.pi * (r ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.133954
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math.pi * (r ** 2)
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:10) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.183454
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:14) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.147294
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:15) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.152218
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:16) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.213973
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:16) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.306811
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:16) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.381927
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:16) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.537439
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:16) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.604429
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:16) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.240721
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:16) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.163234
-- ERROR:
sh: 1: cannot open /home/codebench/codes/1590/825/cbtemp.txt: No such file

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 16:57:17) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.14314
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:02:30) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area, 3))
print(round(volume, 3))


-- EXECUTION TIME:
0.183227
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:13:14) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi * (r1 ** 2)
volume = 4 / 3 * math.pi * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.175728
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:17:26) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r1 = float(input("r1: "))
r2 = float(input("r2: "))
area = math.pi() * (r1 ** 2)
volume = 4 / 3 * math.pi() * (r2 ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.188078
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    r2 = float(input())
EOFError: EOF when reading a line

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:18:55) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = float(input("raio: "))
area = math.pi() * (r ** 2)
volume = 4 / 3 * math.pi() * (r ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.140592
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math.pi() * (r ** 2)
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:19:33) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = float(input("raio: "))
area = math.pi * (r ** 2)
volume = 4 / 3 * math.pi * (r ** 3)
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.128659
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math.pi * (r ** 2)
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:22:00) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = float(input("raio: "))
area = math.pi * r ** 2
volume = 4 / 3 * math.pi * r ** 3
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.146846
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math.pi * r ** 2
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:39:56) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = float(input("raio: "))
area = math(pi) * r ** 2
volume = 4 / 3 * math(pi) * r ** 3
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.162644
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math(pi) * r ** 2
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:39:59) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = float(input("raio: "))
area = math(pi) * r ** 2
volume = 4 / 3 * math(pi) * r ** 3
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.126114
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math(pi) * r ** 2
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:39:59) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
r = float(input("raio: "))
area = math(pi) * r ** 2
volume = 4 / 3 * math(pi) * r ** 3
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.203079
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 5, in <module>
    area = math(pi) * r ** 2
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:45:33) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
from macth import*
r = float(input("raio: "))
area = math.pi * r ** 2
volume = 4 / 3 * math.pi * r ** 3
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.214854
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 4, in <module>
    from macth import*
ImportError: No module named 'macth'

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:46:09) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
from math import*
r = float(input("raio: "))
area = math.pi * r ** 2
volume = 4 / 3 * math.pi * r ** 3
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.158984
-- ERROR:
Traceback (most recent call last):
  File "/home/codebench/codes/1590/825/main.py", line 6, in <module>
    area = math.pi * r ** 2
NameError: name 'math' is not defined

*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
== SUBMITION (2019-03-31 17:46:25) 
-- CODE:
# Teste seu codigo aos poucos.
# Nao teste tudo no final, pois fica mais dificil de identificar erros.
# Nao se intimide com as mensagens de erro. Elas ajudam a corrigir seu codigo.
from math import*
r = float(input("raio: "))
area = pi * r ** 2
volume = 4 / 3 * pi * r ** 3
print(round(area,3))
print(round(volume,3))


-- EXECUTION TIME:
0.170302
-- TEST CASE 1:
---- input:
4.5

---- correct output:
63.617
381.704
---- user output:
63.617
381.704
-- TEST CASE 2:
---- input:
12.34
---- correct output:
478.388
7871.076
---- user output:
478.388
7871.076
-- GRADE:
100%
*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*
